backend/Services/SightingService.cs

```diff
@@ -54,5 +55,9 @@ public Sighting CreateSighting(CreateSightingRequest request)
 
             return _sightings.CreateSighting(newSighting);
         }
+        public Sighting GetSightingById(int sightingId)
```

Comment:
Gap between methods:
```suggestion
        }
        
        public Sighting GetSightingById(int sightingId)
```